core/runtime/src/main/java/io/quarkiverse/langchain4j/RegisterAiService.java

```diff
@@ -262,4 +272,16 @@ public ImageModel get() {
             throw new UnsupportedOperationException("should never be called");
         }
     }
+
+    /**
+     * Default toolProvider that does not provide any tools
+     */
+    @ApplicationScoped
```

Comment:
This should definitely not be added. If you follow my suggestion about using the supplier and the implementation does pretty much what is done for `AuditService`, then this problem will go away